### PR TITLE
Add a src parameter to the install button URL

### PR DESCRIPTION
**Pull request by kumar303**
Fixes https://github.com/mozilla/addons-frontend/issues/3618

### CODE REVIEW

File: src/core/installAddon.js [hunk 1]
```diff
@@ -223,11 +226,12 @@ const userAgentOSToPlatform = {
  *
  * type FindInstallUrlParams = {|
  *   installURLs: $PropertyType<AddonType, 'installURLs'>,
+ *   src?: string,
  *   userAgentInfo: UserAgentInfoType,
  * |};
```
Comment (willdurand) [inline src/core/installAddon.js]: Why is this in a comment btw?

Comment (kumar303) [reply src/core/installAddon.js]: Because converting this file to Flow would be an arduous task

**Comment by kumar303** [comment]
>  the src query parameter should not be added if there is no value

Done.

**Comment by kumar303** [comment]
Thanks for the review!

**Pull request merged**